tool/tbot/botfs/fs_other.go

```diff
@@ -85,12 +85,14 @@ func Write(path string, data []byte, symlinksMode SymlinksMode) error {
 	return nil
 }
 
+//nolint:staticcheck // staticcheck does not like our nop implementations.
 // VerifyACL verifies whether the ACL of the given file allows writes from the
 // bot user.
 func VerifyACL(path string, opts *ACLOptions) error {
 	return trace.NotImplemented("ACLs not supported on this platform")
 }
```

Comment:
```suggestion
func VerifyACL(path string, opts *ACLOptions) error {
	if false {
		return nil
	}
	return trace.NotImplemented("ACLs not supported on this platform")
}
```
I think you can trick the linter like this. 

`//nolint:staticcheck` is also good as long as we remember clean them later. Thanks for fixing!